actfw/command_server.py

```diff
@@ -44,6 +53,7 @@ def __init__(self, sock_path=None):
         self.img = None
 
     def run(self):
+        """Run activity"""
```

Comment:
Run activity って言いますかね？ 英語として不自然な気が。
↓とか？
```suggestion
        """Run and start the activity"""
```

Reply:
fix